src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -21,6 +21,13 @@
 #include "core/common/device.h"
 #include <boost/optional/optional.hpp>
 #include <boost/property_tree/json_parser.hpp>
+#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/join.hpp>
+#include <boost/lexical_cast.hpp>
+#include <boost/thread.hpp>
+#include <boost/asio.hpp>
+#include <boost/asio/signal_set.hpp>

```

Comment:
Let's not add a dependency on a new boost libraries.  The boost::thread library is replaced with std::thread.  If you are looking for some functionality that appears to be missing in std::thread vs boost::thread, then please reach out.

Reply:
Taken care.. Removed this..